website/events/views.py

```diff
@@ -248,3 +249,20 @@ def get(self, request, *args, **kwargs):
             raise Http404("There is no upcoming event.")
 
         return redirect(upcoming_activity)
+
+
+class ICalHelpView(TemplateView):
+    """Render the iCal feed help page."""
+
+    template_name = "events/ical_help.html"
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context["all_events_feed"] = settings.BASE_URL + reverse("events:ical-en")
+        if self.request.member:
+            token = FeedToken.objects.get_or_create(member=self.request.member)[0].token
+            context[
+                "personal_feed"
```

Comment:
`personal_feed` was previously a template tag; we should either delete the code for the tag, or use that instead of manually setting context here.